test/gurka/test_phonemes.cc

```diff
@@ -450,6 +449,7 @@ TEST(Standalone, PhonemesWithAltandDirection) {
           name_pronunciations.find(name_index);
       if (iter == name_pronunciations.end()) {
         if (name_index != 0)
+          // Whats the point of this? this will always be true
```

Comment:
@gknisely there are many places in this test where we are doing such checks. The following `EXPECT_EQ` never fails. Can we check for specific `name_index` instead for which we expect the pronunciation to **not** exist?

I've left comments in other places below for similar conditions. Let me know what you think?